qdao-node/exo-pallet/src/lib.rs

```diff
@@ -168,6 +172,10 @@ pub mod pallet {
             _hash: T::Hash,
             _result: Vec<u8>,
```

Comment:
Please remove _ from used arguments. The origin will need to be a member of the approved auditors after #30 and that is when the origin will be checked.